LongWarOfTheChosen/Localization/LW_PerkPack_Integrated/XComGame.int

```diff
@@ -91,9 +91,9 @@ LocPromotionPopupText="<Bullet/> Cannot be used on the same turn as Run and Gun.
 LocFriendlyName="Close Combat Specialist"
 LocFlyOverText="Close Combat Specialist"
 ; LWOTC Needs Translation
-LocLongDescription="During enemy turns, fire a free reaction shot with your <Ability:WeaponName/> at any visible enemy within four tiles who moves or fires."
+LocLongDescription="Fire a free reaction shot with your <Ability:WeaponName/> at any visible enemy within four tiles who moves or fires."
 ; End Translation
-LocHelpText="During enemy turns, Fire a free reaction shot at any visible enemy within four tiles who moves or fires."
+LocHelpText="Fire a free reaction shot at any visible enemy within four tiles who moves or fires."
```

Comment:
Since CCS is working on XCOM's turn right now and the actual changes to how CCS work is beyond the scope and my ability, I at least could match the description. If CCS were to made working according to previous description, then the description could be reverted.

Reply:
OK, let's keep the change. The localisation can be updated if and when the behaviour of Close Combat Specialist is changed.